pom.xml

```diff
@@ -512,5 +512,7 @@
         <commons-io-version>2.4</commons-io-version>
         <enforcer-plugin-version>1.4</enforcer-plugin-version>
         <failsafe-plugin-version>2.19.1</failsafe-plugin-version>
+
+        <java.version>1.8</java.version>
```

Comment:
I'm not sure that removing support for JRE 1.6 and JRE 1.7 should be done in a minor release.